include/verilated_vcd_c.h

```diff
@@ -97,6 +97,9 @@ class VerilatedVcd {
     vluint64_t m_wrChunkSize;  ///< Output buffer size
     vluint64_t m_wroteBytes;  ///< Number of bytes written to this file
 
+    std::vector<char> m_suffixes;  ///< VCD line end string codes + metadata
+    char* m_suffixesp;  ///< Pointer to first element of above
```

Comment:
```suggestion
    const char* m_suffixesp;  ///< Pointer to first element of above
```

And when build it versus use it, use m_suffixes instead.